src/posts/posts.controller.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Delete,
+  HttpCode,
+  HttpStatus,
+  Param,
+  ValidationPipe,
+} from '@nestjs/common';
+import { PostsService } from './posts.service';
+import { CreatePostDto, UpdatePostDto } from './dto/post.dto';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { PostModel } from './entities/post.entity';
+
+@ApiTags('posts')
+@Controller('posts')
+export class PostsController {
+  constructor(private postsService: PostsService) {}
+
+  @Post('create')
+  @HttpCode(HttpStatus.CREATED)
+  @ApiOperation({ summary: 'Create new post' })
+  @ApiResponse({
+    status: 201,
+    description: 'The post has been successfully created.',
+  })
+  @ApiResponse({ status: 403, description: 'Forbidden.' })
+  async create(@Body(new ValidationPipe()) createPostDto: CreatePostDto) {
+    this.postsService.create(createPostDto);
+  }
+
+  @Get()
+  @ApiOperation({ summary: 'Get all posts' })
+  async getPosts(): Promise<PostModel[]> {
```

Comment:
```suggestion
  async getPosts() {
```

This one don't needed cause you gonna to expose to frontend. I mean nobody can't use controller methods